data/avro-schema/src/main/resources/avro/net/corda/data/p2p/markers/LinkManagerDiscardedMarker.avsc

```diff
@@ -0,0 +1,18 @@
+{
+  "type": "record",
+  "name": "LinkManagerDiscardedMarker",
+  "namespace": "net.corda.p2p.markers",
+  "fields": [
+    {
+      "name": "message",
```

Comment:
was a bit on the fence between adding the full message or just the message ID. Settled on the former thinking that it would be useful for investigation purposes given these are invalid messages, which shouldn't be too frequent.

Reply:
Fair enough.